decoder/Cargo.toml

```diff
@@ -31,11 +31,11 @@ defmt-json-schema = { version = "0.1", path = "./defmt-json-schema" }
 
 # elf2table
 anyhow = "1.0.65"
-gimli = { version = "0.29", default-features = false, features = [
+gimli = { version = "0.31", default-features = false, features = [
     "read",
     "std",
 ] }
-object = { version = "0.35", default-features = false, features = [
+object = { version = "0.36", default-features = false, features = [
```

Comment:
Unless we rely on a feature in 0.36, we don't need this. Cargo will resolve to the latest anyway and there's no use on over-specifying the dependencies in case users have conflicting requirements.